05-script.md

```diff
@@ -440,3 +440,34 @@ Of course, this introduces another tradeoff between flexibility and complexity.
 > # Script 3
 > echo $@.dat
 > ~~~
+
+> ## Ooops my large script has a bug. How can I easily find the problem? {.challenge}
+>
+> You have written a large script with many lines, and it's not working properly.
+> How can I easily identify the problem ?
```

Comment:
That's OK Marcel. I wasn't offended, the comments were aimed at improving the language. :)